### PR TITLE
Erica Andersen's Grumblr Homework

### PR DESCRIPTION
**Pull request by ericacomfort**
Comfort: 4/5
Completeness: 5/5

Concerns: I was stuck for a little on why `this.model.comments.create(data)` could be set in `grumble.js` view to create a comment, yet `this.model.destroy();` had to be set in the `comment.js` view to delete a comment. I tried to come up with a solution to find the id of a grumble's comment in the `grumble.js` view but in the scope I could only seem to find a way to find the id of a grumble. 
**TLDR:** Is it possible in the scope of this `grumble.js` view to find a particular grumble's comment id?

### CODE REVIEW

**Comment by RobertAKARobin** [comment]
Good question! There has to be some logic about Comments in the Grumble model because the Grumble view includes some comments. We should be reviewing this more today.
